app/assets/javascripts/_common.js

```diff
@@ -62,6 +62,32 @@ $(document).ready(function() {
      $('#filter_container').toggle('fast');
    });
 
+  if ($('#Use_Bulk_Note')) {
+    var bulk_note_box = $('#Use_Bulk_Note');
+    var bulk_place = $('#bulk-note-place');
+    bulk_place.hide()
+    var row_notes = $('.single-note-place')
+    var order_form = bulk_note_box.parents('form:first')
+
+    bulk_note_box.change(function(e){
+      if (bulk_note_box[0].checked === true){
+        bulk_place.show();
+        row_notes.hide();
+      } else if (bulk_note_box[0].checked === false) {
+        bulk_place.hide();
+        row_notes.show();
+      }
+    })
+
+    order_form.submit(function(e){
+      e.preventDefault
+      if (bulk_note_box[0].checked === true){
+        row_notes.parents('tr').find(".toggle").filter(":checked").parents("tr").find('.single-note-place').find("input").val(bulk_place.find("input").val())
```

Comment:
I'm curious why you decided to go this route.  There's an existing pattern for bulk updating reconciliation date and for consistency I would vote for using the same approach (check out `OrderDetails::Reconciler`) but there might be something I'm missing here.